wsd/ClientSession.cpp

```diff
@@ -2856,7 +2862,7 @@ void ClientSession::handleTileInvalidation(const std::string& message,
     int normalizedViewId = getCanonicalViewId();
 
     std::vector<TileDesc> invalidTiles;
-    if((part == _clientSelectedPart && mode == _clientSelectedMode) || _isTextDocument)
+    if(part == _clientSelectedPart || _isTextDocument)
```

Comment:
are we sure? it was introduced 2 years ago in commit 7043365 together with "mode", some time before our new approach was implemented (wids and dark mode)

it was working fine and condition seems to be correct. I think we did some change recently, would be good to bisect and find real cause

Reply:
btw we should get rid of `mode` and use the new thing probably in the future :)